#167190530 Implement User Sign Out

ayodejiAA opened the pull request:
#### What does this PR do?
For a user to properly sign out of the application, the user 's token in the session table is set to inactive to prevent future access to any protected route.

#### Description of Task proposed in this pull request?
- Create destroy session route (signout route) 
**GET**: ```/api/v1/sessions/destroy```
- Add an integration test
- Add route to swagger documentation

#### How should this be manually tested (Quality Assurance)?
- Check out this branch ```ft-implement-user-signout-167190530``` and add the database connection string to your environment variable.
- Use any route testing app to test the route. With any token from the session seeds, add it to the headers 

#### What are the relevant pivotal tracker stories?
- Story ID [#167190530](https://www.pivotaltracker.com/story/show/167190530)

#### Any background context you want to add (Operations Impact)?
- ```npm install``` to install necessary dependencies.
- Add a database connection string to env file
- Access route with **GET**: ```/api/v1/sessions/destroy```

#### What I have learned working on this feature:
-  I learnt more on sequelize and working from PR branches.

#### Screenshots: 
<img width="1124" alt="Screenshot 2019-08-02 at 1 37 15 AM" src="https://user-images.githubusercontent.com/42908597/62336015-3fe4e300-b4c6-11e9-8619-718c3539ebf9.png">

dinobi commented on test/users/signout.test.js:
```diff
@@ -0,0 +1,36 @@
+  });
+});
+
+export default chai;
```
why export chai?

dinobi replied:
please also change this file name to session test

ayodejiAA replied:
Okay. I will rename the file.

ayodejiAA replied:
I did that so I could import to the folder index file. I believe it would still work without exporting chai. So I will remove it.

dinobi commented on test/users/signout.test.js:
```diff
@@ -0,0 +1,36 @@
+    expect(response).to.have.status(200);
+  });
+
+  it('respond 500 error if if there is an error', async () => {
```
typo `if if`

dinobi commented on test/users/signout.test.js:
```diff
@@ -0,0 +1,36 @@
+      })
+    };
+    await destroy({}, res);
+    sinon.assert.calledOnce(next);
```
why is signout calling `next`

ayodejiAA replied:
I called next to confirm that the spy function was called once.

dinobi reviewed (changes requested):
Update your PR

ayodejiAA committed:
feature-[167190530]: Implement User Signout

Add a signout route to set token to inactive
Add an integration test
Add feature to swagger documentation
[Delivers #167190530]

chialuka reviewed (dismissed):
Good job

ayodejiAA commented:
> Update your PR

Done.

dinobi commented on test/users/sessions.test.js:
```diff
@@ -118,3 +118,45 @@ describe('LOGIN SERVER ERROR TEST', () => {
+      .request(app)
+      .get('/api/v1/sessions/destroy')
+      .set('authorization', sessionToken);
+    expect(response).to.have.status(200);
```
please also add the response message

ayodejiAA replied:
I have added it.

dinobi reviewed (dismissed):
Please minor comment above
